config/markets.ts

```diff
@@ -702,6 +702,66 @@ const config: {
       thresholdForStableFunding: decimalToFloat(5, 2), // 5%
       thresholdForDecreaseFunding: decimalToFloat(0), // 0%
     },
+    {
+      tokens: { indexToken: "SHIB", longToken: "WETH", shortToken: "USDC" },
+      virtualTokenIdForIndexToken: hashString("PERP:SHIB/USD"),
+      virtualMarketId: hashString("SPOT:SHIB/USD"),
+
+      ...synthethicMarketConfig, // FIXME or synthethicMarketConfig_IncreasedCapacity ?
+
+      positionImpactExponentFactor: decimalToFloat(22, 1), // 2.2
+
+      negativePositionImpactFactor: decimalToFloat(5, 10), // 0.05% for ~45,000 USD of imbalance
+      positivePositionImpactFactor: decimalToFloat(25, 11), // 0.05% for ~90,000 USD of imbalance
+
+      positiveMaxPositionImpactFactor: decimalToFloat(5, 3), // 0.5%
+      negativeMaxPositionImpactFactor: decimalToFloat(5, 3), // 0.5%
+
+      positiveSwapImpactFactor: decimalToFloat(25, 10), // 2.5e-9
+      negativeSwapImpactFactor: decimalToFloat(5, 9), // 5e-9
+
+      swapFeeFactorForPositiveImpact: decimalToFloat(5, 5), // 0.05%
+      swapFeeFactorForNegativeImpact: decimalToFloat(7, 5), // 0.07%
```

Comment:
could remove swapFeeFactorForPositiveImpact and swapFeeFactorForNegativeImpact, i don't think it is there in the other market configs

Reply:
yep seems like it's only used in swapOnly markets